lib/features/guide/widgets/about_us/about_us_tab.dart

```diff
@@ -0,0 +1,94 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../../../api_base/directus_assets_url.dart';
+import '../../../../config/ui_config.dart';
+import '../../../../utils/context_extensions.dart';
+import '../../../../utils/where_non_null_iterable.dart';
+import '../../../../widgets/my_error_widget.dart';
+import '../../../study_circle_details/widgets/details_screen_app_bar.dart';
+import '../../../study_circle_details/widgets/details_screen_sliver_header_section.dart';
+import 'models/link_data.dart';
+import 'models/member_data.dart';
+import 'repository/about_us_repository.dart';
+import 'widgets/desription_section.dart';
+import 'widgets/links_section.dart';
+import 'widgets/section_header.dart';
+import 'widgets/team_section.dart';
+
+class AboutUsTab extends StatelessWidget {
+  const AboutUsTab({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: DetailsScreenAppBar(context, title: context.localize.guide),
+      body: const _AboutUsView(),
+    );
+  }
+}
+
+class _AboutUsView extends ConsumerWidget {
+  const _AboutUsView();
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final state = ref.watch(aboutUsRepositoryProvider);
+
+    return switch (state) {
+      AsyncLoading() => const CircularProgressIndicator(),
+      AsyncError(:final error) => MyErrorWidget(error),
+      AsyncValue(:final value) => CustomScrollView(
+          slivers: [
+            SliverPersistentHeader(
+              delegate: SliverHeaderSection(
+                logoImageUrl: AboutUsConfig.defaultLogoUrl,
+                backgroundImageUrl:
+                    value?.aboutUs?.cover?.filename_disk?.directusUrl,
+              ),
+            ),
+            SliverList(
+              delegate: SliverChildListDelegate(
+                [
+                  SectionHeader(text: context.localize.about_us),
+                  DescriptionSection(text: value?.aboutUs?.description ?? ''),
+                  SectionHeader(text: context.localize.meet_out_team),
+                  TeamSection(
+                    members: value?.aboutUsTeam?.whereNonNull
+                            .toList()
```

Comment:
actually this `toList` shouldn't be here, as we want one single `toList` at the end of this chain of `iterable` operations like `whereNonNull`, `map`, `where`, etc. Cause all of them are lazily evaluated and `toList` actually makes the iteration and convert all this condition to list in a single operation

if you know what i mean. We use basically use `toList` as a final step, and if we only want to make a simple single iteration, we may not even need it at all

Reply:
also my `whereNonNull` can be evoked on nullable type as well and has built-in empty list default.

so you can change `aboutUsTeam?.whereNonNull` to `aboutUsTeam.whereNonNull` and you can delete ` ?? List.empty()` then

Reply:
when i tried to remove `?` i got an error about nullable list

Reply:
I'm almost sure `whereNonNull` is an extension on both nullable and nonnullable lists